tests/bor/mining_test.go

```diff
@@ -14,6 +14,9 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with Erigon. If not, see <http://www.gnu.org/licenses/>.
 
+// Exclude when running tests with race detector see issue #15007
```

Comment:
Can this one just be skipped on macOS instead? I saw that was the only platform that fails, and it seems to be more conservative with the race detector @mh0lt